src/main/events/library/open-game.ts

```diff
@@ -21,8 +24,31 @@ const openGame = async (
 
   if (fs.existsSync(gamePath)) {
     const setupPath = path.join(gamePath, "setup.exe");
+    console.log(setupPath);
     if (fs.existsSync(setupPath)) {
-      shell.openExternal(setupPath);
+      if (process.platform === "win32") {
+        shell.openExternal(setupPath);
+      } 
+      if (process.platform === "linux") {
+        if (spawnSync('which', ['lutris']).status === 0) {
```

Comment:
you can change this for `switch/case` or add `elseif` in line 32 for the best and peformed pratice. In cases when `proccess.platform` is `"win32"` check if `proccess.platform` is `"linux"` is unecessary 😄